### PR TITLE
add to update university and email in profile

### PR DESCRIPTION
**Pull request by willkoua**
| Q                                                      | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                      | [Features]
| Tickets (_issues_) concerned               | [TV-166]

---

## What have you changed ?
Modification du component edit profile pour permettre la modification de l'email et de l'université

## How did you change it ?
en créant un nouveaut component (profile-activate) et en modifiant le component (profile-edit)

### success message
image (1)

### error message
image

## Verification :
 -  [X] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [X] This Pull-Request fully meets the requirements defined in the issue
 -  [ ] I added or modified the attached tests
 

[TV-166]: https://fjnr-inc.atlassian.net/browse/TV-166

### CODE REVIEW

**Comment by willkoua** [comment]
> Please give some screenshot to help understand this PR

screenshot is done

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -700,6 +700,10 @@
   "users-pages.filter.do_not_filter": "Ne pas filtrer",
   "users-pages.filter.labels.membership": "Type de membership",
   "users-pages.users": "Utilisateurs",
+  "validate.email.hello": "Bonjour",
+  "validate.email.messageError": "Votre modification n'a pas pu être effectué. Si le problème persiste, merci de contacter notre administration avec l'email ci-dessous.",
+  "validate.email.messageSuccess": "Votre modification a été faite avec succes.",
```
Comment (RignonNoel) [inline src/assets/i18n/fr.json]: Accent sur succes

File: .gitignore [hunk 2]
```diff
@@ -42,5 +42,4 @@ src/assets/i18n/strings.json
 .DS_Store
 Thumbs.db
 
-# translate file generate
```
Comment (RignonNoel) [inline .gitignore]: This line was goo, let it in peace :)

File: src/assets/i18n/fr.json (hunk 1 above)
Comment (willkoua) [reply src/assets/i18n/fr.json]: done

File: .gitignore (hunk 2 above)
Comment (willkoua) [reply .gitignore]: done

**Pull request merged**